### PR TITLE
Fix rounded corners on institutions logos [OSF-6741]

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

Allows institutions with rounded edges to be displayed properly.

## Changes

- Moved logos into new; banner, shield and shield-rounded-corners folders.
- Created script to make logo images larger and replace alpha with white.
- Changed institution model to support generating links for both square and rounded corners.
- Created test html to view rounded images. (fig 1)

## Side effects

none that I know of.

## Ticket

https://openscience.atlassian.net/browse/OSF-6741

(fig 1)
image

### CODE REVIEW

File: scripts/image_maniplation/size insitutions_images_for_circle.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    newSize = int(pythagonreanTheorem(img.size[0],img.size[1])) # new corners within circle
+    newImg = Image.new('RGBA', (newSize,newSize), 'white')
+    newImg.paste(img, (newSize/2-img.size[0]/2,newSize/2-img.size[1]/2)) #center in larger img
+    return newImg
+
+def alphaToWhite(img):
+    pixeldata = list(img.getdata())
+    for i,pixel in enumerate(pixeldata):
+        if len(pixel) == 4:
+            if pixel[3] == 0:
```
Comment (HalcyonChimera) [inline scripts/image_maniplation/size insitutions_images_for_circle.py]: How does this work with images or logos that may have non-zero opacity? i.e. if it's semi-transparent?

Comment (Johnetordoff) [inline scripts/image_maniplation/size insitutions_images_for_circle.py]: We politely ask the institution for a better logo? :} good find.

File: scripts/image_maniplation/size insitutions_images_for_circle.py [hunk 2]
```diff
@@ -19,8 +19,8 @@ def alphaToWhite(img):
     pixeldata = list(img.getdata())
     for i,pixel in enumerate(pixeldata):
         if len(pixel) == 4:
-            if pixel[3] == 0:
-                pixeldata[i] = (255,255,255,255)
+            if pixel[3] != 255:
+               pixel[3] = 255
```
Comment (HalcyonChimera) [inline scripts/image_maniplation/size insitutions_images_for_circle.py]: Noice.

**Comment by HalcyonChimera** [comment]
Looks good.

**Comment by brianjgeiger** [comment]
Works for me. 🎉 🐧 🎉

**Pull request merged**